development/init/elasticsearch/init.sh

```diff
@@ -27,8 +27,74 @@ init_index() {
 
   echo "Initializing index ${INDEX_NAME}..."
 
-  curl --silent -XPUT "${ELASTICSEARCH_URL}/${INDEX_NAME}"
-  curl --silent -XPUT "${ELASTICSEARCH_URL}/${INDEX_NAME}/_mapping" -d "@/init/mappings/${INDEX_NAME}.json"
```

Comment:
I would prefer if we could load the mapping from the json file instead of copying it plain-text here, so we only need to modify the mapping in one place when we want to make changes to it. This this syntax to read the file via parameter not work for you?